### DISCUSSION TITLE
search strategy when using exact vector search

### DISCUSSION THREAD

**Discussion by kiki00kiki** [qdrant/qdrant#4200] [Q&A] [answered]
If the query conditions include precise( exact: true ) vector search and the search of indexed fields in the payload, what is the strategy for executing the query?

Does it first query the payload index, filter out the vectors that meet the conditions, and then calculate their similarity?

**Comment by generall** [comment, marked as answer]
> Does it first query the payload index, filter out the vectors that meet the conditions, and then calculate their similarity?


yes

**Reply by kiki00kiki** [reply]
thanks
